### ISSUE TITLE
Segmentation fault in tf.quantization.quantize_and_dequantize

### ISSUE THREAD

**Issue by mjkim720** [tensorflow/tensorflow#42105]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 18.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): v2.2.0-rc4-8-g2b96f3662b 2.2.0
- Python version: 3.7.6
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: N/A
- GPU model and memory: N/A

**Describe the current behavior**
`tf.quantization.quantize_and_dequantize` produces a segfault  when `input` is a tensor in any shape of `float32` or `float64` and `axis` is specified to a large number. 

**Describe the expected behavior**
No segfault

**Standalone code to reproduce the issue**
```python
import tensorflow as tf
tf.quantization.quantize_and_dequantize(input=[2.5, 2.5], input_min=[0,0], input_max=[1,1], axis=10)
```

**Other info / logs** Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached.
`Segmentation fault (core dumped)`

**Comment by mihaimaruseac** [comment]
Reopening since #42109 seems to not be solving this issue (this op is V2, the PR fixes on V1)

**Comment by lvyuqi** [comment]
Does the CVE-2020-15265 Vulnerability Affect 1.15.5?

**Comment by mihaimaruseac** [comment]
Yes, but we no longer patch 1.15. Please update post 2.1

**Comment by mgmm13** [comment]
Hello, is there any chance that vulnerability fix will be applied in versions like 2.3.4?

**Comment by mihaimaruseac** [comment]
Hi. Not in 2.3.4, as the 2.3.x has reached end of life.

It is already included in 2.4.0 and later

**Comment by mgmm13** [comment]
@mihaimaruseac noted on this, thank you for the feedback.
